virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,14 +1,14 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the default pod network
 include::_attributes/common-attributes.adoc[]
 :context: virt-using-the-default-pod-network-with-virt
 
 toc::[]
 
 You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
```

Comment:
"You can use the default pod network"? haha. That's like saying "You can use the law of gravity." :-)

I would change this to the default pod network sentence in your other assembly (By default, CNV has a single, default, internal pod network). I know this is outside the scope of your task, so take it or leave it.

Then I would change the binding method to make it more prescriptive. 

You don't have to do this. If you decide not to, let me know and I'll create a DDF.

Reply:
No prob. Changed to:

By default, {VirtProductName} is installed with a single, internal pod network. To use it, bind with `masquerade` mode. Do not use `masquerade` mode with non-default networks.

Yeah, personally I use "you" sparingly. Unfortunately we have a few writers who think if you don't say "You.... blah blah" it's not in active tense. Bizarre, but true. I once had to change the RNs because EVERY sentence started with YOU!!!!! AUGGHHH